.prettierignore

```diff
@@ -0,0 +1 @@
+generators/app/templates/service/package.json
```

Comment:
Also here all files need to have empty lines at the end. VSCode has a setting for this